src/utils/getUserById.tsx

```diff
@@ -0,0 +1,5 @@
+import usersFromServer from '../api/users';
+
+export function getUserById(todoId: number) {
```

Comment:
[CODE STYLE] - Don't use one-letter variable naming. It would be clearer to name the parameter 'userId' instead of 'todoId', as it refers to the ID of a user, not a todo.